modules/mono/glue/Managed/Files/Vector2.cs

```diff
@@ -132,6 +132,13 @@ public Vector2 CubicInterpolate(Vector2 b, Vector2 preA, Vector2 postB, real_t t
                                 (-p0 + 3.0f * p1 - 3.0f * p2 + p3) * t3);
         }
 
+        public Vector2 DirectionTo(Vector2 b)
+        {
+            var v = new Vector2(b.x - x, b.y - y);
+            v.Normalize();
+            return v;
```

Comment:
Would be better as a oneliner: `return new Vector2(b.x - x, b.y - y).Normalized();`

Reply:
Yeah, I did it first but thinked its too cost uneffecient to CPU. Ok changed it back